test/e2e/httpproxy/include_exact_condition_test.go

```diff
@@ -36,8 +36,7 @@ func testIncludeExactCondition(namespace string) {
 		)
 
 		for _, ns := range []string{appNamespace, adminNamespace} {
-			f.CreateNamespace(ns)
-			defer f.DeleteNamespace(ns, false)
+			defer f.NamespacedTest(ns, nil)
```

Comment:
we need to make sure this test is called with the NamespacedTest helper with the correct namespaces for it to create/delete

we can then get rid of this whole loop (here and other places) since the NamespacedTest helper should be actually calling this test function and doing the create/delete before/after the test for us